packages/application-components/package.json

```diff
@@ -27,35 +27,35 @@
     "README.md"
   ],
   "dependencies": {
-    "@babel/runtime": "^7.19.0",
-    "@babel/runtime-corejs3": "^7.19.0",
+    "@babel/runtime": "^7.20.13",
+    "@babel/runtime-corejs3": "^7.20.13",
     "@commercetools-frontend/application-shell-connectors": "21.23.6",
     "@commercetools-frontend/assets": "21.23.6",
     "@commercetools-frontend/constants": "21.23.6",
     "@commercetools-frontend/i18n": "21.23.6",
     "@commercetools-frontend/l10n": "21.23.6",
-    "@commercetools-uikit/card": "^15.11.0",
-    "@commercetools-uikit/constraints": "^15.11.0",
-    "@commercetools-uikit/design-system": "^15.11.0",
-    "@commercetools-uikit/flat-button": "^15.11.0",
-    "@commercetools-uikit/icon-button": "^15.11.0",
-    "@commercetools-uikit/icons": "^15.11.0",
-    "@commercetools-uikit/messages": "^15.11.0",
-    "@commercetools-uikit/primary-button": "^15.11.0",
-    "@commercetools-uikit/secondary-button": "^15.11.0",
-    "@commercetools-uikit/secondary-icon-button": "^15.11.0",
-    "@commercetools-uikit/spacings": "^15.11.0",
-    "@commercetools-uikit/text": "^15.11.0",
-    "@commercetools-uikit/utils": "^15.11.0",
-    "@emotion/react": "11.10.4",
-    "@emotion/styled": "11.10.4",
+    "@commercetools-uikit/card": "^15.11.2",
+    "@commercetools-uikit/constraints": "^15.11.2",
+    "@commercetools-uikit/design-system": "^15.11.2",
+    "@commercetools-uikit/flat-button": "^15.11.2",
+    "@commercetools-uikit/icon-button": "^15.11.2",
+    "@commercetools-uikit/icons": "^15.11.2",
+    "@commercetools-uikit/messages": "^15.11.2",
+    "@commercetools-uikit/primary-button": "^15.11.2",
+    "@commercetools-uikit/secondary-button": "^15.11.2",
+    "@commercetools-uikit/secondary-icon-button": "^15.11.2",
+    "@commercetools-uikit/spacings": "^15.11.2",
+    "@commercetools-uikit/text": "^15.11.2",
+    "@commercetools-uikit/utils": "^15.11.2",
+    "@emotion/react": "11.10.5",
```

Comment:
Since we are using version `11.10.4` in `ui-kit` and you warned about potential issues derived from using mixed versions in the same project, I was wondering if we should consider updating the dependency here.

![image](https://user-images.githubusercontent.com/97907068/218982289-e2d8aa8f-9113-4f34-af30-7ac24cab5d52.png)

Reply:
That's fine, we have the version range so we can force the update the latest version here (see https://github.com/commercetools/merchant-center-application-kit/pull/2960/commits/f8a198c3ffface4c6c268d20c5192ecc3a03a2e5)